From d4b81d01be377be8c5e04525e05c1a8ee2bb3758 Mon Sep 17 00:00:00 2001
From: Aymen Zayet <aymen.zayet@intel.com>
Date: Tue, 10 Sep 2013 19:16:30 +0200
Subject: [PORT FROM MAIN][BT] libbt: take into account _next used in TARGET_PRODUCT.

BZ: 137499

Change-Id: I6087d3074de519589351370d35b07ee689591fa4
Orig-Change-Id: I39c2a75edb0fe1bef8e14419ad08c26c38b878d1
Category: device enablement
Domain: CWS.BT-Common
Origin: internal
Upstream-Candidate: no, not-aosp
Signed-off-by: Aymen Zayet <aymen.zayet@intel.com>
Reviewed-on: http://android.intel.com:8080/131171
Reviewed-by: Combes, Patrick <patrick.combes@intel.com>
Reviewed-by: Gros, Julien <julien.gros@intel.com>
Reviewed-by: Fiat, Christophe <christophe.fiat@intel.com>
Tested-by: Champciaux, Nicolas <nicolas.champciaux@intel.com>
---
 conf/intel/Android.mk             |   10 ++++++++++
 conf/intel/baylake/Android.mk     |    4 ++++
 conf/intel/byt_m_crb/Android.mk   |    4 ++++
 conf/intel/byt_t_ffrd8/Android.mk |    4 ++++
 conf/intel/ctp7160/Android.mk     |    4 ++++
 conf/intel/redhookbay/Android.mk  |    4 ++++
 6 files changed, 30 insertions(+), 0 deletions(-)

diff --git a/conf/intel/Android.mk b/conf/intel/Android.mk
index ed63796..e0bab45 100644
--- a/conf/intel/Android.mk
+++ b/conf/intel/Android.mk
@@ -4,6 +4,7 @@ include $(CLEAR_VARS)
 
 LOCAL_MODULE := bt_vendor.conf
 LOCAL_MODULE_TAGS := optional
+ifeq (, $(filter %_next, $(TARGET_PRODUCT)))
 LOCAL_REQUIRED_MODULES := \
 	bt_baylake.conf \
 	bt_byt_m_crb.conf \
@@ -11,6 +12,15 @@ LOCAL_REQUIRED_MODULES := \
 	bt_ctp7160.conf \
 	bt_redhookbay.conf \
 	bt_saltbay.conf
+else
+LOCAL_REQUIRED_MODULES := \
+	bt_baylake_next.conf \
+	bt_byt_m_crb_next.conf \
+	bt_byt_t_ffrd8_next.conf \
+	bt_ctp7160_next.conf \
+	bt_redhookbay_next.conf \
+	bt_saltbay_next.conf
+endif
 
 include $(BUILD_PHONY_PACKAGE)
 
diff --git a/conf/intel/baylake/Android.mk b/conf/intel/baylake/Android.mk
index 57ceaba..8047a7c 100644
--- a/conf/intel/baylake/Android.mk
+++ b/conf/intel/baylake/Android.mk
@@ -1,7 +1,11 @@
 LOCAL_PATH:= $(call my-dir)
 include $(CLEAR_VARS)
 
+ifeq (, $(filter %_next, $(TARGET_PRODUCT)))
 LOCAL_MODULE := bt_$(notdir $(LOCAL_PATH)).conf
+else
+LOCAL_MODULE := bt_$(notdir $(LOCAL_PATH))_next.conf
+endif
 $(call print-vars $(LOCAL_MODULE))
 LOCAL_MODULE_TAGS := optional
 LOCAL_MODULE_CLASS := ETC
diff --git a/conf/intel/byt_m_crb/Android.mk b/conf/intel/byt_m_crb/Android.mk
index 57ceaba..8047a7c 100644
--- a/conf/intel/byt_m_crb/Android.mk
+++ b/conf/intel/byt_m_crb/Android.mk
@@ -1,7 +1,11 @@
 LOCAL_PATH:= $(call my-dir)
 include $(CLEAR_VARS)
 
+ifeq (, $(filter %_next, $(TARGET_PRODUCT)))
 LOCAL_MODULE := bt_$(notdir $(LOCAL_PATH)).conf
+else
+LOCAL_MODULE := bt_$(notdir $(LOCAL_PATH))_next.conf
+endif
 $(call print-vars $(LOCAL_MODULE))
 LOCAL_MODULE_TAGS := optional
 LOCAL_MODULE_CLASS := ETC
diff --git a/conf/intel/byt_t_ffrd8/Android.mk b/conf/intel/byt_t_ffrd8/Android.mk
index 57ceaba..8047a7c 100644
--- a/conf/intel/byt_t_ffrd8/Android.mk
+++ b/conf/intel/byt_t_ffrd8/Android.mk
@@ -1,7 +1,11 @@
 LOCAL_PATH:= $(call my-dir)
 include $(CLEAR_VARS)
 
+ifeq (, $(filter %_next, $(TARGET_PRODUCT)))
 LOCAL_MODULE := bt_$(notdir $(LOCAL_PATH)).conf
+else
+LOCAL_MODULE := bt_$(notdir $(LOCAL_PATH))_next.conf
+endif
 $(call print-vars $(LOCAL_MODULE))
 LOCAL_MODULE_TAGS := optional
 LOCAL_MODULE_CLASS := ETC
diff --git a/conf/intel/ctp7160/Android.mk b/conf/intel/ctp7160/Android.mk
index 57ceaba..8047a7c 100644
--- a/conf/intel/ctp7160/Android.mk
+++ b/conf/intel/ctp7160/Android.mk
@@ -1,7 +1,11 @@
 LOCAL_PATH:= $(call my-dir)
 include $(CLEAR_VARS)
 
+ifeq (, $(filter %_next, $(TARGET_PRODUCT)))
 LOCAL_MODULE := bt_$(notdir $(LOCAL_PATH)).conf
+else
+LOCAL_MODULE := bt_$(notdir $(LOCAL_PATH))_next.conf
+endif
 $(call print-vars $(LOCAL_MODULE))
 LOCAL_MODULE_TAGS := optional
 LOCAL_MODULE_CLASS := ETC
diff --git a/conf/intel/redhookbay/Android.mk b/conf/intel/redhookbay/Android.mk
index 57ceaba..8047a7c 100644
--- a/conf/intel/redhookbay/Android.mk
+++ b/conf/intel/redhookbay/Android.mk
@@ -1,7 +1,11 @@
 LOCAL_PATH:= $(call my-dir)
 include $(CLEAR_VARS)
 
+ifeq (, $(filter %_next, $(TARGET_PRODUCT)))
 LOCAL_MODULE := bt_$(notdir $(LOCAL_PATH)).conf
+else
+LOCAL_MODULE := bt_$(notdir $(LOCAL_PATH))_next.conf
+endif
 $(call print-vars $(LOCAL_MODULE))
 LOCAL_MODULE_TAGS := optional
 LOCAL_MODULE_CLASS := ETC
-- 
1.7.4.1

